src/pg_timestampz.erl

```diff
@@ -12,11 +13,14 @@
 init(_Opts) ->
     {[<<"timestamptz_send">>], []}.
 
+
+encode({{Year, Month, Day}, {Hour, Minute, Second}, UtcOffset}, TypeInfo) ->
+    encode({{Year, Month, Day}, {Hour + UtcOffset, Minute, Second}}, TypeInfo);
```

Comment:
I feel like this is the simplest possible change to get this supported. However, I don't know if it is the ideal change.

Would you ever want to know the UtcOffset **after** you store it? I don't think that is even possible because these are converted into UTC when stored on the database.

Reply:
PG stores the timestamp in UTC along with the zone. It's useful and common to query and get back the zone information.

Reply:
Yup! I was able to confirm that with wireshark.